main.tf

```diff
@@ -7,50 +23,41 @@ terraform {
   }
 }
 
-data "aws_vpc" "selected" {
-  id = var.vpc_id
+locals {
+  default_tags = {
+    managed-by: "terraform"
+    terraform-module: "terraform-aws-eks"
+  }
+  tags = merge(local.default_tags, var.tags)
 }
 
-data "aws_eks_cluster" "cluster" {
-  name = module.eks.cluster_id
-}
+data "aws_caller_identity" "iam" {}
+data "aws_region" "current" {}
 
-data "aws_eks_cluster_auth" "cluster" {
-  name = module.eks.cluster_id
-}
 
-provider "kubernetes" {
-  version = "~>1.11"
 
-  host                   = data.aws_eks_cluster.cluster.endpoint
-  cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
-  token                  = data.aws_eks_cluster_auth.cluster.token
-  load_config_file       = false
+/*
```

Comment:
Let's get rid of the commented-out code. :-)

Reply:
Not, done yet ;)

Reply:
Yeah, I didn't notice the WIP until it was too late 🌊